README.md

```diff
@@ -83,7 +83,7 @@ Ideal starting point for advanced projects.
 ## Requirements
 - [Node](https://nodejs.org) `6.x` or newer
 - [NPM](https://npmjs.com/) `4.0.0` or newer
-- [Cocoapods](https://cocoapods.org) `1.4.x` or newer
+- [Cocoapods](https://cocoapods.org) `1.5.3` or newer
```

Comment:
@pavjacko is this correct?

Reply:
yep. while it can work with lower.. it's better to avoid lock diffs